src/reactor.rs

```diff
@@ -76,9 +78,10 @@ impl Reactor {
                 events: Mutex::new(Vec::new()),
                 timers: Mutex::new(BTreeMap::new()),
                 timer_ops: ConcurrentQueue::bounded(1000),
+                #[cfg(loom)]
+                mock_events: ConcurrentQueue::unbounded(),
```

Comment:
It seems `cfg(loom)` is unrelated.

Reply:
Sorry, that was a leftover of an attempt to solve #78